cmd/activator/main.go

```diff
@@ -39,14 +39,21 @@ import (
 	"github.com/knative/serving/pkg/activator"
 	activatorhandler "github.com/knative/serving/pkg/activator/handler"
 	activatorutil "github.com/knative/serving/pkg/activator/util"
+	v1alpha12 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
```

Comment:
if you are going to alias this, I'd go with `servingv1alpha1`.

Reply:
I dropped the alias.